code improvements/exercise1.js

```diff
@@ -0,0 +1,5 @@
+function calculateAverage(numbers) {
+  const sum = numbers.reduce((a, b) => a + b, 0);
+  const count = numbers.length;
+  return sum / count;
```

Comment:
Here I think `numbers.length` is understandable and i can be used immediately without creating a variable to hold its value unless it was being use in multiple places